docs/content/latest/develop/gitdev/codespace.md

```diff
@@ -23,8 +23,8 @@ Codespaces doesn't require anything in your local workstation other than a code
 ## Getting Started with a boot app
 You can find the source at [Spring Boot todo on GitHub](https://github.com/srinivasa-vasu/todo).
 
-- The easy way to get started with Codespaces is to simply fork this source repo and jump on to the [Setting up the Codespace environment](#setting-up-the-codespace-environment) section to launch the Codespaces environment for your forked repository. 
-- If you want to set this up from scratch, then follow the below instructions to bootstrap the base project template and copy the appropriate files/content from the [source repo]((https://github.com/srinivasa-vasu/todo)).
+- The easy way to get started with Codespaces is to simply fork this [source repo](https://github.com/srinivasa-vasu/todo) and jump on to the [Setting up the Codespace environment](#setting-up-the-codespace-environment) section to launch the Codespaces environment for your forked repository. 
```

Comment:
I really don't want to include stuff from anyone's personal GitHub repos. Can you please make a Yugabyte-owned repo for this, @srinivasa-vasu? (Jeff is the contact person for that)

Reply:
ping @jharveysmith